resources/views/layouts/partials/header.blade.php

```diff
@@ -25,10 +25,12 @@
 								]
 							) !!}
 						</li>
-						@if(in_array(Route::currentRouteName(), ['project_path', 'sprint_path', 'sprint_live_path', 'snapshot_path']) && isset($sprint))
+						@if(in_array(Route::currentRouteName(), ['project_path', 'sprint_path', 'sprint_live_path', 'snapshot_path']) && isset($project))
 							<li>{!! link_to_route('create_sprint_path', 'New sprint', isset($project) ? $project->slug : $sprint->project->slug) !!}</li>
-							<li><a id="project-settings" href="#" data-toggle="modal" data-target="#project-settings-modal">Project settings</a></li>
-							<li><a id="sprint-settings" href="#" data-toggle="modal" data-target="#sprint-settings-modal">Sprint settings</a></li>
+                            <li><a id="project-settings" href="#" data-toggle="modal" data-target="#project-settings-modal">Project settings</a></li>
+                            @if(isset($sprint))
+                                <li><a id="sprint-settings" href="#" data-toggle="modal" data-target="#sprint-settings-modal">Sprint settings</a></li>
+                            @endif
```

Comment:
well, lines 30-33 in the new code seem to be indented with spaces instead of tabs, so I'd rather use the tab indentation here and that should fix the problem, I believe